study.md

```diff
@@ -66,7 +66,7 @@ Which ActiveRecord method finds all of the records of a certain type (or
 entity)?
 
 ```md
-<!-- your response here -->
+.where
```

Comment:
This will work if there aren't any filtering arguments.  Ideally, you would use `#all`.